src/demo/DemoTestActivitiesService.java

```diff
@@ -154,6 +155,27 @@ public static void testGetActivitiesWithPaginationForClient() throws MambuApiExc
 		printMambuEntityAndActivitiesDetails(mambuEntity, entityId, jsonActivities, fromDate.getTime(), toDate.getTime());
 	}
 
+	/*
+	 * Tests getting ALL activities paginated
+	 */
+	public static void testGetAllActivitiesPaginated() throws MambuApiException{
+		
+		String methodName = new Object() {}.getClass().getEnclosingMethod().getName();
+		System.out.println(methodName = "\nIn " + methodName);
+		
+		ActivitiesService activitiesService = MambuAPIFactory.getActivitiesService();
+		
+		Calendar toDate = Calendar.getInstance(); 
+
+		Calendar fromDate = Calendar.getInstance();
+		fromDate.add(Calendar.DAY_OF_MONTH, -10);
+		
+		List<JSONActivity> jsonActivities = activitiesService.getActivities(fromDate.getTime(), toDate.getTime(),
```

Comment:
My point was to test the method `public List<JSONActivity> getActivities(Date fromDate, Date toDate, Integer offset, Integer limit) throws MambuApiException` from ActivitiesService, the one with only four parameters, because it is public and can be used by our partners. Indeed, behind the scenes it calls the overloaded method, sending two null values for mambuEntity and entityId, respectively.